packages/safe-ds-eda/src/components/profiling/ProfilingInfo.svelte

```diff
@@ -0,0 +1,116 @@
+<script lang="ts">
+    import type { Profiling, ProfilingDetail } from '../../../types/state';
+
+    export let profiling: Profiling;
+    export let imageWidth: number = 200;
+
+    const getProfilingItemColor = function (profilingItem: ProfilingDetail) {
+        if (profilingItem.interpretation === 'good') {
+            return 'var(--primary-color)';
+        } else if (profilingItem.interpretation === 'error') {
+            return 'var(--error-color)';
+        } else if (profilingItem.interpretation === 'warn') {
+            return 'var(--warn-color)';
+        } else if (profilingItem.interpretation === 'bold') {
+            return 'var(--font-dark)';
+        } else if (profilingItem.interpretation === 'default') {
+            return 'var(--font-light)';
+        } else if (profilingItem.interpretation === 'category') {
+            return 'var(--font-light)';
+        } else {
+            return 'var(--font-light)';
+        }
```

Comment:
A `switch` would reduce noise here, since all branches immediately return a value, so no `break` is needed.